oopsypad/server/models.py

```diff
@@ -331,23 +369,54 @@ class Issue(mongo.Document):
 
     reason = fields.StringField()
 
+    location = fields.StringField()
+
     total = fields.IntField(default=1)
 
     meta = {'ordering': ['-total']}
 
+    @property
+    def last_seen(self):
+        minidumps = Minidump.objects(
+            product=self.product,
+            version=self.version,
+            platform=self.platform,
+            crash_reason=self.reason,
+            crash_location=self.location
+        ).only('date_created').order_by('-date_created').first()
+        return minidumps.date_created
+
+    @property
+    def avg_uptime(self):
+        minidumps = Minidump.objects(
```

Comment:
Please use https://docs.mongodb.com/manual/reference/operator/aggregation/avg/ to calculate avg